core/src/main/scala/akka/persistence/postgres/tag/TagTables.scala

```diff
@@ -1,18 +1,19 @@
 package akka.persistence.postgres.tag
 
+import akka.persistence.postgres.config.TagsTableConfiguration
+
 trait TagTables {
 
   import akka.persistence.postgres.db.ExtendedPostgresProfile.api._
 
+  def tagsTableCfg: TagsTableConfiguration
+
   class EventTagTableDefinition(_tableTag: Tag)
-    extends Table[EventTag](
-      _tableTag,
-      _schemaName = None,
-      _tableName = "event_tag") {
+      extends Table[EventTag](_tableTag, _schemaName = tagsTableCfg.schemaName, _tableName = tagsTableCfg.tableName) {
     def * = (id, name) <> (EventTag.tupled, EventTag.unapply)
 
     val id: Rep[Int] = column[Int]("id", O.AutoInc)
```

Comment:
I think that we can also add parameters for id column

Reply:
👍